tools/python/descriptions/setup.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+
+try:
+    # for pip >= 10
+    from pip._internal.req import parse_requirements
+except ImportError:
+    # for pip <= 9.0.3
+    from pip.req import parse_requirements
+import setuptools
+
+module_dir = os.path.abspath(os.path.dirname(__file__))
+sys.path.insert(0, os.path.join(module_dir, "..", "..", ".."))
+from pyhelpers.setup import chdir
+from pyhelpers.setup import get_version
+
+with chdir(os.path.abspath(os.path.dirname(__file__))):
+    setuptools.setup(
+        name="omim-descriptions",
+        version=str(get_version()),
+        author="My.com B.V. (Mail.Ru Group)",
+        author_email="dev@maps.me",
+        description="This package is a description library. "
```

Comment:
a library that provides descriptions (such as those from Wikipedia) to geographic objects